README.md

```diff
@@ -1,3 +1,64 @@
+# Pomodoro Project
+
+<img src="https://user-images.githubusercontent.com/69774407/119630519-b11e2480-be17-11eb-8f89-f1179db1948e.png" width="100" height="100">
+
+## The Pomodoro Technique is a time management method.
+#### The technique uses a timer to break down work into intervals, traditionally 25 minutes in length, separated by short breaks. Each interval is known as a pomodoro, from the Italian word for 'tomato', after the tomato-shaped kitchen timer that Cirillo used as a university student.
+
+## Features of this site:
+- It contains a 25-minute timer (pomodoro period).
+- Contains a pause button to timer.
+- When the timer ends, five minutes are displayed with the start button.(short break).
+- When the short break finishes (5 min) Will come back to (pomodoro period).
+
+
+## How to run it?
+#### requires [Node.js](https://nodejs.org/en/download/) Latest LTS Version: 14.17.0 to run.
+```
+cd git clone https://github.com/the-sol/pomodoro.git
+```
+```
+npx start
+```
```

Comment:
Add an empty line before the beginning of each code block. Also, you should combine these 2 blocks of code into a single block